### PR TITLE
Kit

### PR DESCRIPTION
**Pull request by LeanderK**
- added KIT-Logo
 - converted H&S and talKIT to black
 - rearranged H&S and talKIT logos

### CODE REVIEW

File: src/css/stylesheet.css [hunk 1]
```diff
@@ -13,6 +13,7 @@
 }
 
 #home {
+    background-image: url("/img/bg.png");
```
Comment (jundl77) [inline src/css/stylesheet.css]: Why do we need this? This is already in the bg class

Comment (LeanderK) [reply src/css/stylesheet.css]: well something went wrong during the rebase. I didn't introduce it deliberately

Comment (jundl77) [reply src/css/stylesheet.css]: You shall not pass!

Comment (jundl77) [reply src/css/stylesheet.css]: Okay, @LeanderK can you remove it real quick?

**Pull request merged**